sqldelight-gradle-plugin/src/test/fixtures/keyword_identifiers/src/main/sqldelight/com/sample/Test.sq

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE test (
+  "DESC" TEXT,
```

Comment:
interesting...should we allow quoted identifiers that arent keywords? It would be pretty easy and could clean up the grammar a loooooot